scripts/qesap/lib/cmds.py

```diff
@@ -371,9 +371,15 @@ def cmd_ansible(configure_data, base_project, dryrun, verbose, destroy=False, pr
         if dryrun:
             print(' '.join(command['cmd']))
         else:
-            ret, _ = lib.process_manager.subprocess_run(**command)
+            ret, out = lib.process_manager.subprocess_run(**command)
             log.debug("Ansible process return ret:%d", ret)
+            playbook_path = command['cmd'][4]
+            playbook_name = os.path.splitext(os.path.basename(playbook_path))[0]
+            log_filename = f"ansible.{playbook_name}.log.txt"
+            log.debug("Write %s getcwd:%s", log_filename, os.getcwd())
+            with open(log_filename, 'w', encoding='utf-8') as log_file:
```

Comment:
I think here `'w'` is fine and we do not need `'a'` as we are writing a different file for each different playbook :+1:

Reply:
why printing `os.getcwd()` is needed?